backend/apps/core/cache/signals.py

```
@@ -2,11 +2,28 @@
 
 logger = logging.getLogger(__name__)
 
+
+def _dispatch_invalidation(tag: str):
+    try:
+        if getattr(settings, "TESTING", False) or getattr(settings, "CELERY_TASK_ALWAYS_EAGER", True):
+            from apps.core.cache.invalidation import invalidate_tag
+            invalidate_tag(tag)
+        else:
+            invalidate_tag_task.delay(tag)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)